src/module_specifier.rs

```diff
@@ -57,10 +116,19 @@ pub fn resolve_import(
 ) -> Result<ModuleSpecifier, SpecifierError> {
   let url = match ModuleSpecifier::parse(specifier) {
     // 1. Apply the URL parser to specifier.
-    //    If the result is not failure, return he result.
+    //    If the result is not failure, return the result.
     Ok(url) => url,
 
-    // 2. If specifier does not start with the character U+002F SOLIDUS (/),
+    // 2. If specifier is a node specifier without node: scheme, return
+    //    the specifier with node: scheme added.
+    Err(ParseError::RelativeUrlWithoutBase)
+      if is_builtin_node_module(specifier) =>
+    {
+      eprintln!("Warning: Resolving \"{specifier}\" as \"node:{specifier}\". If you want to use a built-in Node module, add a \"node:\" prefix.");
```

Comment:
We should be able to disable this warning via the `--quiet` flag in the CLI. Maybe we could have an additional method in `NpmResolver` for allowing the implementor to log these errors. Alternatively, we could just use the `log` crate in deno_graph. There are some other `eprintln`s that we could replace with the log crate while doing that.